### ISSUE TITLE
Close control on modals/table details should have higher contrast

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#815]
Needs to be AA compliant at a minimum.

<img width="506" alt="screen shot 2018-04-27 at 16 42 45" src="https://user-images.githubusercontent.com/18653/39371601-159e0ed4-4a3a-11e8-8063-7b0322ce5d86.png">

<img width="726" alt="screen shot 2018-04-27 at 16 41 32" src="https://user-images.githubusercontent.com/18653/39371547-ea627b4c-4a39-11e8-808c-665b17030b4c.png">
